scripts/openshiftci-presubmit-e2etests.sh

```diff
@@ -7,6 +7,7 @@ set -x
 
 export CI="openshift"
 export TIMEOUT="30m"
+export GINKGO_VERBOSE_MODE="-v"
```

Comment:
https://github.com/openshift/odo/pull/2145/files#r326596049

Reply:
here it is - https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_odo/2145/pull-ci-openshift-odo-master-v4.1-integration/257/artifacts/integration/container-logs/test.log